--- kazam-1.5.3-orig/data/ui/kazam.ui	2014-08-13 19:31:04.000000000 +0100
+++ kazam-1.5.3/data/ui/kazam.ui	2020-08-22 17:23:44.557777783 +0100
@@ -493,7 +493,6 @@
             <property name="visible">True</property>
             <property name="can_focus">False</property>
             <property name="toolbar_style">both</property>
-            <property name="icon_size">32</property>
           </object>
           <packing>
             <property name="left_attach">0</property>
